src/routes/index.js

```diff
@@ -1,7 +1,10 @@
 const authRoutes = require("./authRoutes");
 const postRoutes = require("./postRoutes");
+const searchRoutes = require("../controllers/searchController");
```

Comment:
Could you please add this in postController? No need to create a new one.
Thanks!